source/_templates/index.html

```diff
@@ -6,7 +6,26 @@
         <div class="jumbotron">
           <div class="container">
             <h1>NCAR Python Tutorial</h1>
-            <p>Let's learn</p>
+            <p>
+                What makes this Python tutorial unique is that is has been designed specifically 
+                to meet the needs of, and feedback from, atmospheric and oceanic scientists 
+                making the transition with the NCAR-wide 
+                <a class="reference external" href = "https://www.ncl.ucar.edu/Document/Pivot_to_Python/"> pivot-to-Python</a>. 
+                In particular, this tutorial should be useful for any scientist who already knows how to program 
+                in some other language but is taking up Python for the first time. By spending the 
+                first day on pure Python without importing any additional packages, our beginner 
```

Comment:
> first day

Do we want to say this?  I think this tutorial should be described as self-paced, which I think means we should drop suggestions that the material is timed in any way.  There is probably other content like this throughout the tutorial.

Reply:
I will change it to first course.

Reply:
"Part 1 is designed to take one work day, but you may move through the content
much slower or more quickly."  This is the only other reference to time.

Reply:
...Ok.  That can be fixed easily, then.

This also suggests, though, that we have text throughout that refers to "Parts," which doesn't really fit the actual structure any more.  It's really just 1 big part, right?

Reply:
...Or, more specifically, there are no numbered parts, anymore.

Reply:
I just removed all references to numbered parts of the tutorial.